crates/bevy_graph/src/algos/bfs.rs

```diff
@@ -0,0 +1,135 @@
+use std::collections::VecDeque;
+
+use hashbrown::HashSet;
+
+use crate::graphs::{keys::NodeIdx, Graph};
+
+/// Implementation of the `BFS` algorythm
```

Comment:
Link to an approachable article for each of these please?

Reply:
Will be done 🫡